docs: add open-collective (#5216)
fastify/fastify: CONTRIBUTING.md

1. A lead maintainer must merge SemVer-major changes in this repository.
1. In case it is not possible to reach consensus in a pull-request, the decision
   is left to the lead maintainer's team.

> 1. Read and adhere to the [EXPENSE POLICY](./EXPENSE_POLICY.md).

Reviewer: I don't think this is useful here. Move it to another section.

Reviewer: @Eomm remove this

Reviewer: ping ^

4. Open a pull request to
   [`fastify/website:HEAD`](https://github.com/fastify/website/pulls) adding
   yourself to the
   [team.yml](https://github.com/fastify/website/blob/HEAD/static/data/team.yml)
   file. This list is also sorted alphabetically so make sure to add your name
   in the proper order. Use your GitHub profile icon for the `picture:` field.
5. The person that does the onboarding must add you to the [npm
   org](https://www.npmjs.com/org/fastify), so that you can help maintaining the
   official plugins.

> 6. Optionally, the person can ask to the Open Collective a admins (lead team) to

Reviewer 2: ```suggestion
6. Optionally, the person can ask to the Open Collective admins (lead team) to
```

>    figure out as a collaborator in the Fastify's Open Collective page.

Reviewer 3: Bit lost as to what this is trying to convey?

Author: Here: https://opencollective.com/fastify at the bottom there is the `Out team` section and you need to be added to be there

> 6. Optionally, the person can ask to the Open Collective admins (lead team) to
>    figure out as a collaborator in the Fastify's Open Collective page.

Reviewer 3: ```suggestion
6. Optionally, the person can be added as an Open Collective member 
   by the lead team.
```
